tests/survey_demo_tests.js

```diff
@@ -0,0 +1,18 @@
+var assert = require('chai').assert;
+var sinon = require('sinon');
+require('../survey.js')
+
+suite('Demo test', function () {
+	setup(function() {
+		sinon.spy(SurveyMan.survey, 'getOptionById');
+	});
+
+	teardown(function() {
+		SurveyMan.survey.getOptionById.restore();
+	});
```

Comment:
@etosch , we may cut this teardown function by using sandbox in sinon. We may implement that in next commit if you want.